packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -132,22 +249,22 @@ class ExpansionPanelList extends StatelessWidget {
       vertical: _kPanelHeaderExpandedHeight - _kPanelHeaderCollapsedHeight
     );
 
-    for (int index = 0; index < children.length; index += 1) {
+    for (int index = 0; index < widget.children.length; index += 1) {
       if (_isChildExpanded(index) && index != 0 && !_isChildExpanded(index - 1))
         items.add(new MaterialGap(key: new _SaltedKey<BuildContext, int>(context, index * 2 - 1)));
 
       final Row header = new Row(
         children: <Widget>[
           new Expanded(
             child: new AnimatedContainer(
-              duration: animationDuration,
+              duration: widget.animationDuration,
               curve: Curves.fastOutSlowIn,
               margin: _isChildExpanded(index) ? kExpandedEdgeInsets : EdgeInsets.zero,
               child: new ConstrainedBox(
                 constraints: const BoxConstraints(minHeight: _kPanelHeaderCollapsedHeight),
-                child: children[index].headerBuilder(
+                child: widget.children[index].headerBuilder(
```

Comment:
Since it's used more than once, you can add:
```dart
final ExpansionPanelRadio child = widget.children[index];
```
above, and use `child.headerBuilder(` here, and below.